SpatialGDK/Documentation/content/game-client-interest-management.md

```diff
@@ -0,0 +1,115 @@
+
+Each query within an `ActorInterestComponent` can consist of:
+
+* a single constraint
+* multiple constraints that you combine using `UOrConstraint`s and `UAndConstraint`s 
+
+The following table lists the possible constraints.
+
+| Constraint | Description |
+| --- | --- |
+| `UOrConstraint` | Satisfied if any of its inner constraints are satisfied. |
+| `UAndConstraint` | Satisfied if all of its inner constraints are satisfied.|
+| `USphereConstraint` | Includes all Actors within a sphere centered on the specified point. |
+| `UCylinderConstraint` | Includes all Actors within a cylinder centered on the specified point. |
+| `UBoxConstraint` | Includes all Actors within a bounding box centered on the specified point. |
+| `URelativeSphereConstraint` | Includes all Actors within a sphere centered on the Actor that has `ActorInterestComponent`. |
+| `URelativeCylinderConstraint` | Includes all Actors within a cylinder centered on the Actor that has `ActorInterestComponent`. |
+| `URelativeBoxConstraint` | Includes all Actors within a bounding box centered on the Actor that has `ActorInterestComponent`. |
+| `UCheckoutRadiusConstraint` | Includes all Actors of a class or a derived class within a cylinder centered on the Actor that has `ActorInterestComponent`. |
+| `UActorClassConstraint` | Includes all Actors of a class. You can optionally include derived classes. |
+| `UComponentClassConstraint` | Includes all Actors with an Unreal Actor component of a specified class. You can optionally include derived classes. |
+
+For more information, see the [interest constraints header file](https://github.com/spatialos/UnrealGDK/blob/0.6.0-rc/SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialInterestConstraints.h).
+
+To attach this component to an Actor, open the Actor's Blueprint and, on the Components tab, select **Add Component**. Actor Interest is in the SpatialGDK section:
+
+![Add component]({{assetRoot}}assets/screen-grabs/game-client-interest-management/add-component.png)
+
+### Example
+
+![Set up interest]({{assetRoot}}assets/screen-grabs/game-client-interest-management/set-up-interest.png)<br>
+_Image: The Interest section of the details panel for `ActorInterestComponent`_
+
+In the screenshot above, `ActorInterestComponent` has two queries, so the game client receives updates about:
+
+* all Pawns within 200 meters of the Actor that has `ActorInterestComponent` 
+* all Actors of the class `Landmark` or derived from the class `Landmark`
+
+## AlwaysInterested
+
+You might want a game client to always receive updates about some Actors, regardless of the positions of the player and of these Actors. For example, you might want the game client to always receive updates about an important landmark, or about other members of the player’s team. You can use the `AlwaysInterested` `UPROPERTY` specifier for this.
```

Comment:
> You might want a game client to always receive updates about some **specific** Actors...

Just to drive in the point that this is about a particular Actor rather than a general type of an Actor.